entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -808,8 +809,19 @@ public void bulkUpdateAllMatchingFilter(
       final BulkUpdateAllMatchingFilterResponse response = doBulkUpdate(request, requestContext);
       responseObserver.onNext(response);
       responseObserver.onCompleted();
+    } catch (final StatusException e) {
+      LOG.warn(
+          String.format(
+              "Error while bulk updating entities with request %s for %s",
+              request, tenantIdOptional.orElse("")),
```

Comment:
instead of logging tenant id, log the context.

Also can use slf4j log string template instead of explicit string format.

Reply:
Oh! That's actually smart. 🙂

Reply:
Continuing with the merge since it's a nit. Will probably piggy back the changes in some subsequent changes.